src/app/import/import-redmine/import-redmine.service.ts

```diff
@@ -59,6 +60,7 @@ export class ImportRedmineService {
   }
 
   importProject(args: ImportRedmineArgs): Observable<TrackForeverProject> {
+    this.fetchService.setBaseUrl(args.serverUrl);
```

Comment:
This should really be an argument to the fetch service, but realistically we won't be injecting this in multiple places so I guess it's fine.

Reply:
That's true. I was in a hurry and decided to just use the existing code.